### DISCUSSION TITLE
iOS Live Photos and .heic format

### DISCUSSION THREAD

**Discussion by robflate** [alangrainger/immich-public-proxy#15] [Feature Requests]
When I use the built in share feature in Immich it shows me iOS Live Photos (.heic) files taken on my iPhone. I can see the full size image and view the short live video. This works in Safari and Chrome.

When using immich-public-proxy, I cannot watch the short live video in either Safari or Chrome. Neither can I see the full size image in Chrome. Instead it says `Oops... Failed to load content...`. I assume this is because Chrome does not support .heic files. Maybe immich-public-proxy should do what Immich does and show a converted file (.jpeg) version.

**Comment by alangrainger** [comment]
> Maybe immich-public-proxy should do what Immich does and show a converted file (.jpeg) version.

Great suggestion - this is now implemented in v1.4.0.

I will leave this open as a feature request for the "short live video" part.

**Reply by TobiasDax** [reply]
for the live video portion it would be great to have this be enabled/ disabled via configuration.

**Comment by Danie10** [comment]
The .heic and live photos are also applicable for Samsung phones. It's my default save format on a S23 Ultra.

**Comment by eddified** [comment]
I am having an issue with download of HEIC files from the gallery.

Details: Single HEIC image is shared.
I enabled Gallery for single image.
Click the Gallery download single image icon. (Not the "download all" button).
The image downloads, _and_ the data is all there, but the filename is "original" with no extension. This makes it not openable for some programs. 

For JPGs, it seems to know the correct extension, and it names it "original.jpg". But for HEICs, just "original". 

Expected: "IMG_1234.HEIC". Actual: "original".

Acceptable, but not as good: "original.heic".

**Comment by Danie10** [comment]
Yes I am noticing the same with an album of photos. If I right-click on the image and save as it converts it to jpg and saves as preview.jpeg. If I did the photo download option it downloads to a filename called original without any extension. It is jpg though. So seems to be an issue with auto converting / naming.
